components/common/AppModal.vue

```diff
@@ -136,10 +102,6 @@ export default {
     nav {
       padding-right: 8px;
     }
-
-    .modal {
-      transform: translateX(-4px);
-    }
   }
 
   @media only screen and (min-width: 984px) {
```

Comment:
question(non-blocking): can't we use predefined tailwind classes like `lg` instead of media queries?